cpp/src/parquet/statistics.cc

```diff
@@ -463,6 +463,8 @@ class TypedStatisticsImpl : public TypedStatistics<DType> {
  public:
   using T = typename DType::c_type;
 
+  // Currently, this function will tend to be called by ColumnWriter to create the
```

Comment:
Nit (and same below)
```suggestion
  // Currently, this constructor will tend to be called by ColumnWriter to create the
```